src/arviz_plots/plots/dist_plot.py

```diff
@@ -244,6 +253,17 @@ def plot_dist(
     else:
         aes_by_visuals = aes_by_visuals.copy()
     aes_by_visuals.setdefault("dist", plot_collection.aes_set.difference("y"))
+    col_dim_list = ["model", "group", "energy", "alpha"]
+    col_dim_list_bool = [dim in distribution for dim in col_dim_list]
+    if any(col_dim_list_bool):
+        aes_by_visuals.setdefault(
+            "face", {"color"}.union(plot_collection.aes_set.difference({"y", "linestyle"}))
+        )
+    else:
+        aes_by_visuals.setdefault(
+            "face", plot_collection.aes_set.difference({"y", "linestyle", "color"})
+        )
```

Comment:
I don't understand the logic here. And hardcoding the `col_dim_list` will become a headache soon. I think mappings should be the same as for "dist" or `plot_collection.aes_set.difference({"y", "linestyle"})`

Reply:
`face_density` not having `chain` means `density` doesn't have chain either but `dist` doesn't have any of this logic and it works. This means that the plots that call `plot_dist` internally are defining `aes_by_visual` to make sure things work. There are probably values being set for `dist` but not for `face` because it didn't exist yet. Let's discuss a bit tomorrow at the meeting, but I see two solutions:

* default to `set(aes_by_visuals["dist"]).difference("linestyle")` (this is right after the setdefault for `dist` key in aes_by_visual so will always exist)
* update the plots that use `plot_dist` to ensure they define something different explicitly when the default won't make sense (note that even if we do option 1 we might still need to make updates to some of the plots)